src/m_menu.c

```diff
@@ -4380,6 +4381,9 @@ setup_menu_t gen_settings4[] = { // General Settings screen4
   {"Level Brightness", S_THERMO|S_STRICT, m_null, M_X_THRM,
    M_Y + gen4_level_brightness*M_SPC, {"extra_level_brightness"}},
 
+  {"Organize save files", S_YESNO|S_PRGWARN, m_null, M_X,
```

Comment:
It's too confusing if we allow to change it in-game, right?

Reply:
Yes, I think so. It would also require more code changes.